src/views/component.njk

```diff
@@ -57,72 +56,59 @@
   <p class="govuk-body">{% block componentDependencies %}{% endblock %}</p>
 
   <h2 class="govuk-heading-l">Installation</h2>
-  <pre><code>npm install --save @govuk-frontend/{{ componentName }}</code></pre>
+
+  <code>npm install --save @govuk-frontend/{{ componentName }}</code>
```

Comment:
What's the logic in `<pre><code>` vs `<code>`? Looking at the rendered readmes in GitHub, these no longer render as code blocks but as inline. Is that the desired outcome?

Reply:
its a single line so doesnt stringly need pre. but can unify it

Reply:
I think if it's a single line but not inline it should still be a code _block_.

I.e. the difference between `this` inline code and

```
this single line code block
```

Reply:
ql, updated